src/components/RecipeItem/RecipeItem.jsx

```diff
@@ -12,7 +12,9 @@ import clockIcon from "../../icons/clockIcon.svg";
 
 const RecipeItem = ({ title, image = defaultPic, cookingTime, dishTypes }) => {
   let cookingTimeStr = "";
-  const [dish = "Not available"] = dishTypes;
+  let dish = "";
+
+  dishTypes !== undefined ? ([dish] = dishTypes) : (dish = "Not available");
```

Comment:
I think you should swap the order of ternary outcomes and write `dishTypes === undefined` as it's more easily readable.